### PR TITLE
Added basic api skeleton

### PR DESCRIPTION
**Pull request by devang-gaur**

### CODE REVIEW

File: src/api/handlers.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var req_validations = require("./validations.js").request_validations()
+
+module.exports.request_handlers = function() {
+	return {
+		get_experiment: function(experiment_id) {
```
Comment (devang-gaur) [inline src/api/handlers.js]: @anmolbabu @mik-dass  Can you suggest what Handler should I add? I think get_experiment is manager-api specific.

**Comment by anmolbabu** [comment]
@dev-gaur 

1. Please rename the repo as measurements-api
2. The api here is supposed to be the ones that scouts use to push data. so you could probably add an a simple POST api /api/v1.0/measurements that just echo's to console whatever is passed it to it in request body

File: src/api/handlers.js (hunk 1 above)
Comment (anmolbabu) [reply src/api/handlers.js]: Please have a look at https://github.com/fabric8-hdd/measure-api/pull/1#issuecomment-359366518
I think ingest data or add_measure or something on those lines would be appropriate here

Comment (mik-dass) [reply src/api/handlers.js]: I think handle_stats_ingestion or something on those lines.

**Comment by devang-gaur** [comment]
@anmolbabu Changes made. Review again pliss.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "name": "measure-api",
+  "version": "1.0.0",
+  "description": "Micro-service for metric measurements and handling of scouts' requests.",
+  "scripts": {
+    "check_lint": "./node_modules/.bin/eslint ./.eslintrc.json",
+    "test": "mocha --timeout 5000 tests/*_test.js --exit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabric8-hdd/measure-api.git"
```
Comment (anmolbabu) [inline package.json]: url might need changes its measurements-api now right?

File: src/api/handlers.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+					if (valid) {
+						let msg = {
+							result: "Success"
+						}
+						resolve(JSON.stringify(msg))
+					} else {
+						reject("Stat " + JSON.stringify(stat) + " is not of supported type")
+					}
+					// ToDo(dev-gaur):
+					// 1. Check if stat is required by an experiment
```
Comment (anmolbabu) [inline src/api/handlers.js]: ToDos here May not now be relevant here with each of them broken down into separate micro-services.
Have a look and remove those that don't fit in here

File: src/api/routes.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+var appRouter = function(app) {
+	app.post("/api/v1.0/measure", function(req , res){
+		console.log("The request body is: ", req.body)
+		let stat = JSON.parse(req.body)
+		req_handlers.handle_stats_ingestion(stat).then(function (result) {
+			res.status(200)
+			res.send(String(result))
+		}).catch(function(err) {
+			console.log("Error "+ err.toString())
+			// ToDo: Fix Http Status code to non 200
```
Comment (anmolbabu) [inline src/api/routes.js]: Please remove this comment

**Review by anmolbabu — COMMENTED** [review]
Verified?

**Comment by devang-gaur** [comment]
@anmolbabu Changes made. Thanks for bearing with me.

**Review by anmolbabu — APPROVED** [review]
LGTM

**Comment by mik-dass** [comment]
LGTM

**Pull request merged**